nifi-frontend/src/main/frontend/apps/nifi/src/app/pages/flow-designer/ui/canvas/items/flow/import-from-registry/import-from-registry.component.ts

```diff
@@ -397,6 +399,7 @@ export class ImportFromRegistry extends CloseOnEscapeDialog implements OnInit {
                 }),
                 disconnectedNodeAcknowledged: this.clusterConnectionService.isDisconnectionAcknowledged(),
                 component: {
+                    name: this.importFromRegistryForm.get('customFlowName')?.value,
```

Comment:
If a user types in a value in the Custom Versioned Flow Name field then removes the value, an empty string is sent to the backend which returns a 400 (see screenshot below). There are a couple of ways you could handle this, but this is the most straightforward IMO.
```suggestion
                    name: this.importFromRegistryForm.get('customFlowName')?.value || null,
```
![Screenshot 2025-05-05 at 11 52 41 AM](https://github.com/user-attachments/assets/dcf0e427-40a4-4184-9448-9e384bccc033)